roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
I was thinking clusters exposed to the Internet, where this would be wrong.  all-in-one is different if you're assuming those are development instances.